File: FireDrill/Javascript/taskEight.js
let sum = 0
for(let count = 1; count < 11;count++){ 
	let multiples = 1
	if(count % 4 == 0){
		for(let counter = 1; counter < 6;counter++){
			multiples = multiples * count 
			sum += multiples
			}
	}
	}
		console.log(sum)

File: FireDrill/Java/Taskeight.java
public class Taskeight{
	public static void main(String [] args){
			int sum = 0;
		for(int count = 1; count < 11;count++){
			int multiples = 1;
				for(int counter = 1; counter < 6; counter++){
			if(count % 4 == 0){
				multiples *= count;
				sum += multiples;
					}
				
				}
				}

			System.out.print(sum);

		}



}